samples/cpp/build_samples_msvc.ps1

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Arguments parsing
+param (
+    [string]$BuildDirectory = "",
```

Comment:
```suggestion
    [string]$BuildDirectory = "$Env:USERPROFILE/Documents/Intel/OpenVINO/openvino_${SamplesType}_samples_build",
```
That allow to remove `$BuildDirectory = if ($BuildDirectory) {$BuildDirectory} else {"$Env:USERPROFILE/Documents/Intel/OpenVINO/openvino_${SamplesType}_samples_build"}`

Reply:
Unfortunately, the `param` section should be at the top of the script so we cannot place the `$SamplesType = (Get-Item $SourceDirectory).Name` before it to use it for default path, I'll try to figure something out.